modules/logger/logger.go

```diff
@@ -99,6 +99,10 @@ func (_ testingT) Logf(t testing.TestingT, format string, args ...interface{}) {
 type terratestLogger struct{}
 
 func (_ terratestLogger) Logf(t testing.TestingT, format string, args ...interface{}) {
+	if tt, ok := t.(helper); ok {
+		tt.Helper()
```

Comment:
Sorry, I'm totally missing some context here. What is this helper thing?

Reply:
See https://golang.org/pkg/testing/#B.Helper. Specifically:

> When printing file and line information, that function will be skipped.

Right now all the logs are outputting `logger.go:66` as the file + linenumber information, which is this function.

Reply:
Ohhhh, neat, didn't know about that! Shame it didn't seem to work :-\